src/user/reducers/index.js

```diff
@@ -7,6 +7,7 @@ const initialState = {
 export function userReducer(state = initialState, action) {
```

Comment:
I think that the initial state should be just an empty object, not an object that has a key of `user`

Reply:
changed, also to authReducer ;)